CHANGELOG.rst

```diff
@@ -22,7 +22,9 @@ Fixed
 
 Added
 ~~~~~
-
+* Add support for a configurable connect timeout for SSH connections as requested in #4715
+  by adding the new configuration parameter ``ssh_connect_timeout`` to the ``ssh_runner``
+  group in st2.conf.
```

Comment:
Please move the record up to `In Development` section of Changelog and also include PR # number.
If you want, it's also good to add a credit in Changelog line.

Reply:
Thanks for the hint and done.